### ISSUE TITLE
Failure to run post setup

### ISSUE THREAD

**Issue by CoolHandSquid** [secw01f/webrecon#3]
![image](https://user-images.githubusercontent.com/42553774/79764542-80ee8300-82f3-11ea-8e04-12a77d67cba6.png)
Possibly an error of default output location.

**Comment by secw01f** [comment]
Hi @CoolHandSquid,

The issue should be taken care of. If you just pull down the repo again, it should be good to go.

Thanks!

**Comment by CoolHandSquid** [comment]
![image](https://user-images.githubusercontent.com/42553774/79780104-6aebbd00-8309-11ea-8fbb-dfd05d2204a2.png)
It's a different issue now, so that is a plus. I am still getting this when I run the build script. Possibly part of the issue.
![image](https://user-images.githubusercontent.com/42553774/79780277-a6868700-8309-11ea-8085-3155144cf829.png)

**Comment by secw01f** [comment]
@CoolHandSquid So the installation seems to have still been successful. The python os module is part of every default python installation so I am not concerned about that. I have fixed the error that you are seeing but can you validate that all of the tools installed properly by running assetfinder -h and validating that the tool installed? If it is installed then this should be taken care of. If not then we will need to do some additional trouble shooting. I will wait for confirmation or additional errors before I close this again.

Thanks for your patience

**Comment by CoolHandSquid** [comment]
![image](https://user-images.githubusercontent.com/42553774/79877410-3257ec00-83ba-11ea-801c-98ec9777405a.png)
Was that what you were looking for?

**Comment by CoolHandSquid** [comment]
![image](https://user-images.githubusercontent.com/42553774/79879146-5288aa80-83bc-11ea-9e04-d7c119810a16.png)

**Comment by CoolHandSquid** [comment]
![image](https://user-images.githubusercontent.com/42553774/79879239-6c29f200-83bc-11ea-967a-4b87da70b1b0.png)

**Comment by secw01f** [comment]
Now that the setup script is fixed. All of the underlying tools should work and the tool should run. For your -s file you will not need to add http:// or https:// as the web host validation portion of the tool will take care of that for you. Just a list of IP's or subdomains is all you need.

Once you validate that everything works for you I will close this ticket.

**Comment by CoolHandSquid** [comment]
![image](https://user-images.githubusercontent.com/42553774/79902730-fc2d6300-83df-11ea-83f1-f6d838a96b9e.png)

**Comment by CoolHandSquid** [comment]
10.10.10.9 is confirmed up.

**Comment by secw01f** [comment]
did you re run the the setup.sh script?

**Comment by CoolHandSquid** [comment]
![image](https://user-images.githubusercontent.com/42553774/79906318-93e18000-83e5-11ea-9c65-b0830f5dcba3.png)
Did it again just now. Same result.

**Comment by secw01f** [comment]
Do you mind running setup.sh, exiting the terminal, restarting the terminal, and then run the go command? Then send me the output.

Thanks again!

**Comment by secw01f** [comment]
@CoolHandSquid 

Good news, I got the setup.sh file straightened out so that all of the tools should install and should work right away. If you are still having issues, we can re-evaluate.

Thanks!

**Comment by CoolHandSquid** [comment]
cloned in this morning and ran it the same as yesterday. Identical result.

**Comment by secw01f** [comment]
I have updated the installation instructions in the README and made a few updates to the setup.sh file. I followed the process on a fresh install of Kali 2020 and it worked when I followed the process so I hope that you get the same results. I will continue working on the setup file so that everything will run without having to restart the terminal. Once you are up and running I will close this issue.

Fingers crossed!
